addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -158,6 +158,7 @@ def _default_company_id(self):
     attachment_number = fields.Integer(compute='_get_attachment_number', string="Number of Attachments")
     employee_name = fields.Char(related='emp_id.name', string="Employee Name")
     attachment_ids = fields.One2many('ir.attachment', 'res_id', domain=[('res_model', '=', 'hr.applicant')], string='Attachments')
+    meeting_ids = fields.Many2many('calendar.event', string="Meeting")
```

Comment:
Instead of adding a `m2m`, add a `applicant_id` field on `calendar.event` and use the same mecanism as in crm module.